Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml

```diff
@@ -122,7 +122,7 @@
                 android:importantForAccessibility="no"
                 app:layout_constraintStart_toStartOf="@id/vaccination_consent_qr_code_icon"
                 app:layout_constraintTop_toBottomOf="@id/vaccination_consent_qr_code_text"
-                app:srcCompat="@drawable/ic_qr_time" />
+                app:srcCompat="@drawable/ic_shield_illustration" />
 
```

Comment:
`marginVertical` for `MoreInformationView` seems a bit to large in the layout when comparing it to figma

Reply:
updated